### ISSUE TITLE
Additional filter functionality

### ISSUE THREAD

**Issue by onetrev** [wp-sync-db/wp-sync-db#24]
Just wondering if it would it be possible to add in these actions and filters that are found in this free add-on to WP Migrate Pro? https://github.com/deliciousbrains/wp-migrate-db-pro-tweaks/

**Comment by slang800** [comment]
Yeah, that should totally be possible. The only thing you need to do is change the function names used in the code so it says "wp-sync-db"... Other than that renaming, the functionality should be identical.
